modxcms/revolution
# Rework concatenated php in lexicon entries

### What does it do?
Replaced concatenation with placeholders in Chunk, TV, and Snippet lexicons, and made coordinating updates to their respective Form Panels (js).

### Why is it needed?
Fixes problem with Crowdin parsing lexicon entries with concatenation, identified by @alroniks [here](https://github.com/modxcms/revolution/pull/16118#discussion_r854314344).

### How to test
Create/edit a chunk, snippet, and TV to ensure the form works as expected and the copyable tag in the name description displays and behaves correctly. Ultimately, the true test will be post-merge once the changed lexicon files get processed via Crowdin.

### Related issue(s)/PR(s)
No posted issue (responding to @alroniks' comment referenced above)
